scripts/linters/webpack_config_linter_test.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for webpack_config_linter.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import multiprocessing
+
+from core.tests import test_utils
+
+from . import pre_commit_linter
+from . import webpack_config_linter
+
+NAME_SPACE = multiprocessing.Manager().Namespace()
+PROCESSES = multiprocessing.Manager().dict()
+NAME_SPACE.files = pre_commit_linter.FileCache()
+FILE_CACHE = NAME_SPACE.files  # pylint: disable=redefined-builtin
+
+
+class AppDevLinterTests(test_utils.GenericTestBase):
```

Comment:
Why is this named as AppDev, any specific reason?

Reply:
Sorry my bad Changed it to WebpackConfigLinterTests